### PR TITLE
fixing ls -la /dev/zero failure

### PR DESCRIPTION
**Pull request by Dawody**

### CODE REVIEW

File: src/fs/driver/devfs/devfs_oldfs.c [hunk 1]
```diff
@@ -133,7 +133,7 @@ static struct inode *devfs_lookup(char const *name, struct inode const *dir) {
 
 	cdevtab = get_cdev_tab();
 	bdevs = get_bdev_tab();
-	for (i = 0; i < MAX_BDEV_QUANTITY; i++) {
+	for (i = 0; i < MAX_BDEV_QUANTITY && bdevs[i]; i++) {
```
Comment (alexkalmuk) [inline src/fs/driver/devfs/devfs_oldfs.c]: But you will break here at the first `bdev[i] == NULL`, and other block devs will not be processed.

Comment (alexkalmuk) [reply src/fs/driver/devfs/devfs_oldfs.c]: It's better to skip null devices while processing all array up to the end

Comment (Dawody) [reply src/fs/driver/devfs/devfs_oldfs.c]: I thought that reaching Null when iterating on the array means that there is no more block devs.
so I added the condition to make it break when reaching Null pointer.

File: src/fs/driver/devfs/devfs_oldfs.c [hunk 2]
```diff
@@ -134,6 +134,8 @@ static struct inode *devfs_lookup(char const *name, struct inode const *dir) {
 	cdevtab = get_cdev_tab();
 	bdevs = get_bdev_tab();
 	for (i = 0; i < MAX_BDEV_QUANTITY; i++) {
+		if(!bdevs[i])
+			continue;
```
Comment (alexkalmuk) [inline src/fs/driver/devfs/devfs_oldfs.c]: Can you please follow our code style - https://github.com/embox/embox/wiki/Code-Style

```
if (...) {
    ...
}
```

File: src/fs/driver/devfs/devfs_oldfs.c (hunk 1 above)
Comment (alexkalmuk) [reply src/fs/driver/devfs/devfs_oldfs.c]: Some of these devices are possibly deleted, so we can skip them, so yes your new checking is pretty fine :)

File: src/fs/driver/devfs/devfs_oldfs.c (hunk 2 above)
Comment (Dawody) [reply src/fs/driver/devfs/devfs_oldfs.c]: I am sorry, I will read your code style to avoid such errors in the future.

**Comment by Dawody** [comment]
@alexkalmuk 
I think that I did something wrong!
could you advise me ?

**Comment by alexkalmuk** [comment]
@Dawody Please squash your commits into a single one and I think it could be merged.

**Pull request merged**